feat: 67_COVID19-start.md (#679)
dialoguemd/covid-19: src/regions/ca/info/faq/67_COVID19-start.md

## Where did COVID-19 start?

> The first reported cases of COVID-19 came from Wuhan, China.

Reviewer: ```suggestion
The first reported cases of COVID-19 originated in Wuhan, China.
```
